pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1507,5 +1507,45 @@ self: super: {
   # 2020-06-24: Tests are broken in hackage distribution.
   # See: https://github.com/robstewart57/rdf4h/issues/39
   rdf4h = dontCheck super.rdf4h;
+  
+  # hasn't bumped upper bounds
+  # test fails: "floskell-test: styles/base.md: openBinaryFile: does not exist (No such file or directory)"
+  # https://github.com/ennocramer/floskell/issues/48
+  floskell = dontCheck (doJailbreak super.floskell);
+
+  # hasn't bumped upper bounds
+  # test fails because of a "Warning: Unused LANGUAGE pragma"
+  # https://github.com/ennocramer/monad-dijkstra/issues/4
+  monad-dijkstra = dontCheck (doJailbreak super.monad-dijkstra);
+
+  # only used by haskell-language-server as of now
+  # Tests disabled: there is an issue with Cabal 3
+  #   -> it builds in `./dist` while it needs `./dist-newstyle` for the tests
+  # not sure if this issue comes from nixpkgs wrapping or lsp-test itself
+  lsp-test_0_11_0_2 = dontCheck super.lsp-test_0_11_0_2;
+
+  # haskell-language-server uses its own fork
+  # Test disabled: it seems to freeze (is it just that it takes a long time ?)
+  hls-ghcide = 
+    dontCheck (
+      super.hls-ghcide.override {
+        # specific versions required
+        lsp-test = self.lsp-test_0_11_0_2;
```

Comment:
Have you confirmed that the dependency override is needed even after disabling tests?

Reply:
Yes